project/src/main/world/environment/grass-overworld-tiler.gd

```diff
@@ -61,6 +64,20 @@ func autotile(value: bool) -> void:
 	_add_random_grass()
 
 
+## Removes all grass tiles.
+func unautotile(value: bool) -> void:
```

Comment:
Revert this 'unautotile' stuff, we'll include this in a later commit.

Reply:
Removed all mentions of 'unautotile' including one introduced in the previous commit (oops)